docs/releases/4.1.1.txt

```diff
@@ -18,3 +18,6 @@ Bugfixes
 * Fixed a regression in Django 4.1 that caused a crash of the ``test``
   management command when running in parallel and ``multiprocessing`` start
   method is ``spawn`` (:ticket:`33891`).
+  
+* Fixed omitting formfield_callback when creating a model form using
+  modelform_factory (:ticket: `31721`).
```

Comment:
This doesn't qualify for a backport based on our [supported versions policy](https://docs.djangoproject.com/en/dev/internals/release-process/#supported-versions), please revert `4.1.1` release notes.

Reply:
Done, thanks! 🙌